actions/notify-workflow-completed/dist/ceda2c0264ac98da5d36.json

```diff
@@ -1,8 +1,8 @@
 {
-    "defaultModule": "github.com/keep-network/keep-core/solidity-v1",
+    "defaultModule": "github.com/keep-network/keep-core/solidity",
```

Comment:
Those generated files look incorrect, aren't they out of sync with the actual source code?